eox_tenant/models.py

```diff
@@ -177,3 +182,13 @@ class Meta:
         Model meta class.
         """
         app_label = "eox_tenant"
+
+class EdnxCourseEnrollmentManager(CourseEnrollmentManager):
+    """
+    Custom manager for CourseEnrollment with Table-level filter methods.
+    """
+    def get_queryset(self):
+
+        orgs_to_exclude = configuration_helpers.get_all_orgs()
```

Comment:
why do you call the get_all_orgs method instead of getting the orgs of the current microsite?